.mergify.yml

```diff
@@ -0,0 +1,17 @@
+pull_request_rules:
+  - name: rebase and merge when passing all checks
+    conditions:
+      - base=master
+      - status-success=continuous-integration/travis-ci/pr
+      - status-success="validate commits"
```

Comment:
Do we plan on enabling the pr-validator for this repo? If so maybe that can go in this PR as well. If not then the `"validate commits"` status can be removed from the mergify config.

Reply:
Great suggestion!  Added the action yml in this PR.